https://src.fedoraproject.org/rpms/t1lib/blob/rawhide/f/t1lib-c99.patch

Add missing parameter types for improved C99 compatibility: implicit
ints are not part of C99.

--- a/lib/type1/regions.c
+++ b/lib/type1/regions.c
@@ -604,6 +604,7 @@
        register struct region *R;  /* region in which we are changing direction */
        fractpel x,y;         /* current beginning x,y                        */
        fractpel dy;          /* direction and magnitude of change in y       */
+       int x2, y2;
 {
        register fractpel ymin,ymax;  /* minimum and maximum Y since last call */
        register fractpel x_at_ymin,x_at_ymax;  /* their respective X's       */
